File: src/Gateways/Api.Gateways.Models/DTOs/Token2FADto.cs


namespace Api.Gateways.Models.DTOs
{ 
    public class Token2FADto
    {
        public int id { get; set; }
        public DateTime creado { get; set; }
        public int idusuario { get; set; }
        public int idaplicacion { get; set; }
        public string token { get; set; }
        public bool aceptado { get; set; }
        public bool rechazado { get; set; }

    }
}

File: Verifica/angularapp/src/app/Models/token2fa.dto.ts
export class Token2FADto {
  id: number;
  creado: Date;
  idusuario: number;
  idaplicacion: number;
  token: string;
  aceptado: boolean;
  rechazado: boolean;

  constructor(
    id: number,
    creado: Date,
    idusuario: number,
    idaplicacion: number,
    token: string,
    aceptado: boolean,
    rechazado: boolean) {
      this.id = id;
      this.creado = creado;
      this.idusuario = idusuario;
      this.idaplicacion = idaplicacion
      this.token = token;
      this.aceptado = aceptado;
      this.rechazado = rechazado;
    }
}

